examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -164,6 +165,27 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- sirdata example -->
```

Comment:
I got the error running this example.

Reply:
There was a quick bug in the check endpoint, I just fixed it so you should be able to test now.